doc/dev/mgmt/tests.md

```diff
@@ -52,14 +52,14 @@ There are several ways to authenticate to Azure, but to be able to record test H
 ### Get a token with Active Directory application and service principal
 
 Follow this detailed tutorial to set up an Active Directory application and service principal:
-https://azure.microsoft.com/documentation/articles/resource-group-create-service-principal-portal/
+https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal
```

Comment:
![image](https://user-images.githubusercontent.com/70930885/164415901-2b1eb7ac-55bb-4ee2-9193-56441ded85e4.png)
now we rely on testproxy so we should not claim we rely on `vcrpy` anymore